cmd/clusterctl/api/v1alpha3/annotations.go

```diff
@@ -25,4 +25,10 @@ const (
 	// Note: Only CRDs that are referenced by core Cluster API CRDs have to comply with the naming scheme.
 	// See the following issue for more information: https://github.com/kubernetes-sigs/cluster-api/issues/5686#issuecomment-1260897278
 	SkipCRDNamePreflightCheckAnnotation = "clusterctl.cluster.x-k8s.io/skip-crd-name-preflight-check"
+
+	// DeleteForMoveAnnotation will be set to Cluster API objects that are going to be
+	// deleted for clusterctl move operation.
+	//
+	// It will help any validation webhook to take decision based on it.
+	DeleteForMoveAnnotation = "clusterctl.cluster.x-k8s.io/delete-for-move"
```

Comment:
Could you kindly document this annotation in https://cluster-api.sigs.k8s.io/reference/labels_and_annotations.html?

Reply:
Sure, added. Thank you!